src/platform/iotjs_module_i2c-linux-general.inl.h

```diff
@@ -134,6 +137,17 @@ int I2cSmbusAccess(int fd, uint8_t read_write, uint8_t command, int size,
 }
 
 
+int I2cSmbusReadByte(int fd) {
+  I2cSmbusData data;
+  int result;
+
+  result = I2cSmbusAccess(fd, I2C_SMBUS_READ, I2C_NOCMD, I2C_SMBUS_BYTE, &data);
+
+  // Mask one byte from result (data.byte).
+  return result >= 0 ? 0xFF & data.byte : -1;
```

Comment:
What is the return value of this function? 0 and -1 are type of `I2cError` or not?